app/pods/agendas/controller.js

```diff
@@ -26,6 +56,35 @@ export default class AgendasController extends Controller.extend(DefaultQueryPar
   successfullyAdded() {
     this.isCreatingNewSession = false;
     this.send('refreshRoute');
-    this.router.transitionTo('agendas.overview');
+    this.router.transitionTo('agendas');
+  }
+
+  @action
+  async onClickRow(agenda) {
+    const meeting = await agenda.createdFor;
```

Comment:
check other comment, createFor should already be loaded for other reasons.

Reply:
Left in the async & await call here because even if we include `createdFor`, `agenda.createdFor` returns a proxy and we'd have to either `await` it or use `meeting.get()`.